types/session_cost.yaml

```diff
@@ -17,6 +74,14 @@ types:
           Absolute timestamp for the end of this chunk in RFC3339 UTC format
         type: string
         format: date-time
+      metervalue_from:
+        description: >-
+          Metervalue (energie Wh import) for the start of this chunk
+        type: integer
```

Comment:
why not using powermeter types here?

Reply:
Because this is as specified in California Pricing Requirements. We just get a metervalue from the CSMS. But if you think we should change this: what is your proposal?

Reply:
> i would use the powermeter type since it would fit better in the ecosystem

I am fine with that, @Pietfried  is that ok for you?